README.md

```diff
@@ -1,2 +1,58 @@
-# hellomicroverse
-In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+   <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> > In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+- Linters
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Editor (Vscode) and Browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
```

Comment:
I am not able to see some of the headers/sections of the readme file. :see_no_evil:  Kindly ensure that each one of them is present here and customised according to your project and credentials. :smile:  This applies to all subsequent projects. :next_track_button:  The only optional header is the FAQ section.